packages/quantic/package.json

```diff
@@ -56,7 +56,7 @@
     "@octokit/graphql-schema": "10.74.2",
     "@salesforce/eslint-config-lwc": "3.3.3",
     "@salesforce/eslint-plugin-lightning": "1.0.0",
-    "@salesforce/sfdx-lwc-jest": "1.1.2",
+    "@salesforce/sfdx-lwc-jest": "1.4.1",
```

Comment:
This change means that the corresponding entry in the `overrides` section of the root `package.json` no longer does anything (and it probably isn't needed anymore). I think we should remove it

Reply:
good catch!